src/pages/kontrollsamtale/KontrollsamtalePage.tsx

```diff
@@ -0,0 +1,103 @@
+        <div className={styles.kontrollsamtalePage}>
+            {RemoteData.fold(
+                () => <Loader />,
+                () => <Loader />,
+                (err) => <ApiErrorAlert error={err as ApiError} />,
+                (kontrollsamtale: Nullable<Kontrollsamtale>) => (
+                    <>
+                        <Heading level="1" size="xlarge">
+                            {formatMessage('kontrollsamtale')}
+                        </Heading>
+                        <BodyLong>
+                            {kontrollsamtale?.innkallingsdato
+                                ? formatMessage('nestePlanlagt') + formatDate(kontrollsamtale.innkallingsdato)
+                                : formatMessage('ingenPlanlagt')}
+                        </BodyLong>
+                        <div className={styles.nyDatoContainer}>
+                            <DatePicker
+                                className={styles.datePicker}
+                                dateFormat="dd.MM.yyyy"
+                                label={formatMessage('velgDatoTittel')}
+                                onChange={(dato: Date | null) => settNyDato(dato)}
+                                value={nyDato ?? toDateOrNull(kontrollsamtale?.innkallingsdato)}
+                                feil={
+                                    (nyDato ?? toDateOrNull(kontrollsamtale?.innkallingsdato)) === null
+                                        ? formatMessage('datovalidering')
+                                        : undefined
+                                }
+                                minDate={iMorgen()}
+                            />
+                            <Button onClick={() => nyDato && handleNyDatoForKontrollsamtaleClick(nyDato)}>
+                                {formatMessage('settNyDato')}
+                                {RemoteData.isPending(nyDatoStatus) && <Loader />}
+                            </Button>
+                        </div>
+                        {RemoteData.isSuccess(nyDatoStatus) && (
+                            <Alert className={styles.alert} variant="info">
+                                {formatMessage('nyDatoBekreftelse')}
+                            </Alert>
+                        )}
+                        {RemoteData.isFailure(nyDatoStatus) && (
+                            <Alert className={styles.alert} variant="error">
+                                {formatMessage('kunneIkkeSetteDato')}
+                            </Alert>
+                        )}
+                    </>
+                )
+            )(nesteKontrollsamtale)}
+            <Button className={styles.tilbakeknapp} variant="secondary" onClick={history.goBack}>
+                <Back />
+                {formatMessage('tilbake')}
+            </Button>
+        </div>
+    );
+};
+
+const iMorgen = (): Date => {
+    const iMorgen = new Date();
+    iMorgen.setDate(iMorgen.getDate() + 1);
+    return iMorgen;
+};
```

Comment:
Bruk gjerne `date-fns` for dato-operationer.

https://date-fns.org/v2.28.0/docs/startOfTomorrow